[VAULT-39627] Add GUI steps for Secret Engine mount tune (#995)
hashicorp/web-unified-docs: content/vault/v1.21.x (rc)/content/docs/commands/secrets/tune.mdx

$ vault secrets tune -audit-non-hmac-request-keys=common_name -audit-non-hmac-request-keys=ttl pki/
```

</Tab>

<Tab heading="GUI" group="gui">

> 

Reviewer: ```suggestion

@include 'alerts/secret-engine-tune-gui-support.mdx'

```

Reviewer: Make sure to include the extra whitespace so the markdown parser doesn't get confused 😅

$ vault secrets tune -audit-non-hmac-request-keys=common_name -audit-non-hmac-request-keys=ttl pki/
```

</Tab>

<Tab heading="GUI" group="gui">

> **Note:** Vault 1.21 supports configuration updates (tune) for mounted **keymgmt** secrets engine in the GUI. Support for other engines may be added in future releases.

> To tune a keymgmt mount:

Reviewer: ```suggestion
To tune a `keymgmt` mount:
```
For plugin types we typically format it as code unless we write the full name out (e.g., Key management)

1. Navigate to **Secrets Engines** in the Vault GUI.

> 1. Select a mounted **keymgmt** secrets engine.

Reviewer: ```suggestion
1. Select a mounted `keymgmt` secrets engine.
```

1. Click **Manage** in the top navigation bar, then select **Configure**.

1. Adjust any of the following settings:
   - **Time-to-live (TTL)** - The default length of time a token or lease is valid for

>    - **Maximum Time-to-live (TTL)** - The maximum allowed lifetime of tokens and leases

Reviewer: ```suggestion
   - **Maximum Time-to-live (TTL)** - The maximum allowed lifetime of tokens and leases.
```

>    - **Description** - A description of this secrets engine mount

Reviewer: ```suggestion
   - **Description** - A human-friendly description of the secrets engine mount.
```
Style correction: avoid "this" as a pronoun
